docker-compose.yml

```
@@ -1,7 +1,7 @@
 services:
   frontend:
     platform: linux/amd64
-    image: umasscds/inatator_frontend:0.1.0
+    image: umasscds/inatator_frontend:0.2.0
```

I know you updated this for testing purposes, but let's not merge it in until you're ready to 'cut a release'.
```suggestion
    image: umasscds/inatator_frontend:0.1.0
```